src/app/components/about/about.component.html

```diff
@@ -1 +1,130 @@
-<p>about works!</p>
+<div class="col-12">
+    <div class="jumbotron text-center bg-danger text-white">
+        <h1 class="display-4 font-weight-bold">Covid-19</h1>
+        <p class="lead">Coronavirus disease (COVID-19) is an infectious disease caused by a newly discovered
+            coronavirus.</p>
+        <hr class="my-4">
+        <p>Most people infected with the COVID-19 virus will experience mild to moderate respiratory illness and recover
+            without requiring special treatment. Older people, and those with underlying medical problems like
+            cardiovascular disease, diabetes, chronic respiratory disease, and cancer are more likely to develop serious
+            illness.</p>
+    </div>
+    <div class="row ">
+        <div class="col-md-12 col-lg-6">
+            <div class="col-md-12 bg-light rounded ">
+                <h4 class="col-12 text-center mb-4 pt-5 text-uppercase">Symptoms</h4>
+                <div class="col-12">
+                    <div class="col">
+                        <div class="row">
+                            <div class="col-12 col-md-4 text-center mb-5">
+                                <div class="col-12 bg-light rounded border border-secondary p-2">
+                                    <img src="../../../assets/5728192 - head headache migraine pain severe shock.png"
+                                        alt="headache">
+                                    <p class="card-text">Headache</p>
```

Comment:
This is not a p, we had this conversation back when we were learning HTML, please refer to that.